lms/djangoapps/save_for_later/api/v1/views.py

```diff
@@ -122,7 +123,8 @@ def post(self, request):
                 {
                     'user_id': user.id,
                     'category': 'save-for-later',
-                    'type': 'course' if course_id else 'program'
+                    'type': 'course' if course_id else 'program',
+                    'send_to_self': True if not user.is_anonymous and user.email == email else False,
```

Comment:
No need to explicitly set True/False, we can simplify this:
```suggestion
                    'send_to_self': not user.is_anonymous and user.email == email,
```